packages/table/src/common/classes.ts

```diff
@@ -17,10 +17,12 @@ export const TABLE_CELL_LEDGER_EVEN = "bp-table-cell-ledger-even";
 export const TABLE_CELL_LEDGER_ODD = "bp-table-cell-ledger-odd";
 export const TABLE_COLUMN_HEADER_TR = "bp-table-column-header-tr";
 export const TABLE_COLUMN_HEADERS = "bp-table-column-headers";
+export const TABLE_COLUMN_HEADER_CELL = "bp-table-column-header-cell";
+export const TABLE_COLUMN_HEADER_CELL_HAS_REORDER_HANDLE = "bp-table-column-header-cell-has-reorder-handle";
```

Comment:
yikes this is pretty long. I would be cool with simply `bp-table-has-reorder-handle` and `bp-table-has-interaction-bar`